src/main/webapp/usermap.jsp

```diff
@@ -15,7 +15,7 @@
     <h1 class="caption">Who is using TEAMMATES?</h1>
     <div id="contentHolder">
         <div id="container" style="position: relative; width: 800px; height: 500px; border: 1px solid #DEDEDE;"></div>
-        <p id="lastUpdate" class="lastUpdate">Last updated: 30 November 2016</p>
+        <p id="lastUpdate" class="lastUpdate">Last updated: <span id="lastUpdateDate" class="lastUpdate"></span></p>
```

Comment:
I didn't remove `p` tag's class because  if i remove it , the test and the date will have different styles is that ok ?I asked the same thing in previous PR but i didn't get any replies from your side.